mula/Makefile

```diff
@@ -109,9 +109,9 @@ endif
 
 itest: ## Run the integration tests.
 ifneq ($(file),)
-	$(ci-docker-compose) run --rm mula python -m pytest tests/integration/${file} $(if $(function),-k $(function))
+	$(ci-docker-compose) run --rm mula sh -c "sleep 10; python -m pytest tests/integration/${file} $(if $(function),-k $(function))"
```

Comment:
I think 10 seconds is quite a lot. Another way to do this is to use health checks (see https://github.com/minvws/nl-kat-coordination/pull/920). I believe this should also work for Postgres containers (see `pg_isready`)

Reply:
Yeah commented above as well to use that as well

Reply:
Fix implemented